package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "marvin",
+  "version": "0.0.0",
+  "description": "An API driven skin for MediaWiki",
+  "scripts": {
+    "start": "run-p -n dev:*",
+    "dev:client": "NODE_ENV=development webpack -w",
+    "dev:server": "NODE_ENV=development nodemon -w dist/client -w src/server/ src/server/index.js",
+    "prod:build": "NODE_ENV=production webpack -p",
+    "format": "prettier --write 'src/**/*.js'",
+    "lint": "eslint --cache --max-warnings 0",
+    "lint:all": "npm run lint -- 'src/**/*.js'",
+    "test": "npm run lint:all",
+    "precommit": "lint-staged"
+  },
+  "lint-staged": {
+    "*.js": ["npm test"]
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/niedzielski/marvin.git"
+  },
+  "keywords": ["mediawiki", "wikimedia", "web", "node", "pwa"],
```

Comment:
Argh, goddammit npm. This was written by `npm init` after filling up the cli stuff...  Needs fixing, I'll add it to TODO.